src/App/App.js

```diff
@@ -17,6 +17,8 @@ import MyNavbar from '../components/MyNavbar/myNavbar';
 import './App.scss';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import authRequests from '../helpers/data/authRequests';
+import smashRequests from '../helpers/data/smashRequests';
```

Comment:
You shouldn't need these requests in this file. They'll happen down where state lives on those article components.

Reply:
👍